site/content/building-apps-for-livechat/extending-server-side.md

```diff
@@ -8,21 +8,36 @@ In some cases you'll need to interact with LiveChat on the server side. We provi
-Customers contact agents via their preferred channels, for example Facebook or Twitter. Agents want a single interface to manage all the incoming messages. When responding, messages would return to the medium it came from.
+Integration of multiple communication channels is a convenient solution for both customers and agents.
+Customers contact agents via their preferred channels, for example Facebook or Twitter. Agents want a single interface to manage all the incoming messages. When responding, messages would return to the medium it came from. With Beta Chat APIs*, you can link any messenging channel with LiveChat.
 
 ### Chatbots\* [beta]
 
-We could take it further and engage chatbots. They could lead conversations with customers when no agents are available. Once agents start accepting chats, they take over the conversations from bots. To make it work, chatbots would need to be trained based on the data from the past conversations.
+Chatbots are multipurpose time savers. They could lead conversations with customers when no agents are available. Once agents start accepting chats, they take over the conversations from bots. To make it work, chatbots would need to be trained based on the data from the past conversations.
 
-\*) We're working on the new set of APIs, which introduce new features and multiple improvements. The most important one is the new Chat API. It allows you to send and receive rich messages, exchange events, and perform all variety of actions within the LiveChat ecosystem. Got you curious? Hit us up at [developers@livechatinc.com](mailto:developers@livechatinc.com) for more details &amp; early access!
+<div class="note">
+*) We're working on the new set of APIs, which introduce new features and multiple improvements. The most important one is the new Chat API. It allows you to send and receive rich messages, exchange events, and perform all variety of actions within the LiveChat ecosystem. Got you curious? Hit us up at <a href="mailto:developers@livechatinc.com">developers@livechatinc.com</a> for more details &amp; early access!
+</div>
 
-## Reporting apps
+## Advanced reporting
 
-Oftentimes companies need customized reports based on data from LiveChat. Those might be very specific collations for the analysis and statistics. For example, the number of chats rated as good in relation to agents' weekly activity. Or, the number of times a particular word appeared in conversations. The key is to pull from the LiveChat database any data that might be needed to create such reports.
+Managers need customized reports based on data from LiveChat. Those might be very specific collations for the analysis and statistics. For example, the number of chats rated as good in relation to agents' weekly activity. Or, the number of times a particular word appeared in conversations. The key is to pull data from the LiveChat via [Reports API](/docs/rest-api/#reports).
```

Comment:
The key is to pull data from the LiveChat via [Reports API](/docs/rest-api/#reports).

from LiveChat (bez the, ogólnie z LiveChata)

Reply:
👍 💯